oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/introspection/NimbusOAuth2TokenIntrospectionClientTests.java

```diff
@@ -254,6 +255,18 @@ public void constructorWhenRestOperationsIsNullThenIllegalArgumentException() {
 				.isInstanceOf(IllegalArgumentException.class);
 	}
 
+	@Test
+	public void setRequestEntityConverterWhenConverterIsNullThenExceptionIsThrown() {
```

Comment:
We should probably have one more test that uses a mock `Converter` to confirm that the setter works.